### PR TITLE
New sentences and small fixes

### LINKED ISSUE #21: Collect new sentences to be added to the bot
Please add possible new or updated sentences for public & private messages in the comments below, let's leave this one open for at least a week, so that @FrederikBolding can do all of these at once. 
If you want to see what the bot is sending currently, please create a new user and enter a room. The gist of it is in [this doc](https://docs.google.com/document/d/1USXi18Fb5_fkNxBRL592t-om_0o16ntpsBijT100ra4/edit?usp=sharing) but has been updated live.

### PR DESCRIPTION
**Pull request by FrederikBolding**
As agreed with @krrisis, fixes #21.

Let me know if there are any issues or questions 😄

### CODE REVIEW

File: chatbot.js [hunk 1]
```diff
@@ -13,85 +13,93 @@ fs.readFile("./privateRooms.json", "utf8", function (err, data) {
...
-          sendNextQuestion(curQuestion, greetingQuestions, user, client, privateRooms[user].welcoming.room);
+        if (positive || negative) {
+          if (greetingQuestions.length > curQuestion + 1) {
+            sendNextQuestion(curQuestion, greetingQuestions, user, client, privateRooms[user].welcoming.room);
+          } else {
+            privateRooms[user].welcoming = undefined;
+          }
         } else {
-          privateRooms[user].welcoming = undefined;
+          sendInternalMessage("I didn't recognize that response :(", user, client);
```
Comment (vojtechsimetka) [inline chatbot.js:64]: is the `(` in the string intentional?

File: pointsbot.js [hunk 2]
```diff
@@ -3,29 +3,35 @@ const { point_types, domains, max_points, sheet_id, sheet_tab_name } = require("
...
 
-exports.handlePointGiving = function(auth, event, room, toStartOfTimeline, client){
+exports.handlePointGiving = function (auth, event, room, toStartOfTimeline, client) {
   if (event.getType() === "m.room.message" && toStartOfTimeline === false) {
     client.setPresence("online");
-    const message = event.getContent().body;
+    let message = event.getContent().body;
     const roomId = room.roomId;
+
+    if (message[1] === " ") {
```
Comment (vojtechsimetka) [inline pointsbot.js:12]: This seems like a little bit strange and unclear condition. At least write a comment what it means.

File: chatbot.js [hunk 3]
```diff
@@ -13,85 +13,93 @@ fs.readFile("./privateRooms.json", "utf8", function (err, data) {
...
-      var question = questionsForRoom[i];
-      var shouldAnswerQuestion = false;
+    for (let i = 0; i < questionsForRoom.length; i++) {
+      let question = questionsForRoom[i];
+      let shouldAnswerQuestion = false;
       if (typeof question.trigger === "string") {
         shouldAnswerQuestion = msg.toLowerCase().includes(question.trigger.toLowerCase());
       } else {
-        for (var t = 0; t < question.trigger.length; t++) {
+        for (let t = 0; t < question.trigger.length; t++) {
```
Comment (vojtechsimetka) [inline chatbot.js:102]: This should probably be some foreach cycle to improve readability.

Also I know that the code below is not yours, but comparing two lower case string is kinda weak. It should be some regex ignoring blank spaces and potentially not perfect matches. I guess this is more feedback for improvement :)

File: chatbot.js [hunk 4]
```diff
@@ -13,85 +13,93 @@ fs.readFile("./privateRooms.json", "utf8", function (err, data) {
...
+}
+
 function checkForRoomQuestions(msg, roomForQuestions, roomToSendIn, user, client) {
-  var questionsForRoom = questions[roomForQuestions];
+  let questionsForRoom = questions[roomForQuestions];
   if (questionsForRoom) {
-    for (var i = 0; i < questionsForRoom.length; i++) {
-      var question = questionsForRoom[i];
-      var shouldAnswerQuestion = false;
+    for (let i = 0; i < questionsForRoom.length; i++) {
```
Comment (vojtechsimetka) [inline chatbot.js:96]: Probably should be foreach cycle

File: chatbot.js (hunk 1 above)
Comment (deamme) [reply chatbot.js:64]: It's just a sad smiley

File: pointsbot.js (hunk 2 above)
Comment (deamme) [reply pointsbot.js:12]: True. It's so we can support "! dish" - requested by Kris

File: chatbot.js (hunk 3 above)
Comment (deamme) [reply chatbot.js:102]: I agree with using forEach - should be done to all for loops in this case. Although a rather minor detail with using regex, which really doesn't matter in this case tbh

**Pull request merged**